Composer/packages/server/src/models/settings/defaultSettingManager.ts

```diff
@@ -19,6 +19,12 @@ export class DefaultSettingManager extends FileSettingManager {
 
   protected createDefaultSettings = (): any => {
     return {
+      feature: {
+        UseTranscriptLoggerMiddleware: false,
+        UseShowTypingMiddleware: false,
+        UseInspectionMiddleware: false,
+        UseCosmosDb: false,
```

Comment:
I think this setting name is ambiguous and confusing.  I had to check if this meant use CosmosDb instead of another persistent storage (e.g. Blob storage) or if it was simply enabling persistent storage in favor of using memory, where CosmosDb is the only option.  

In my opinion, we need to be more explicit here and change this to either **UsePersistentStorage** or **UseCosmosDbPersistentStorage**.